integrations/pinterest-tag/test/index.test.js

```diff
@@ -92,5 +94,20 @@ describe('Pinterest', function() {
         });
       });
     });
+
+    describe('#track', function() {
+      beforeEach(function() {
+        analytics.spy(window, 'pintrk');
+      });
+
+      it('should set Segment messageId as Pinterest Evnet Id', function() {
+        analytics.track('Order Completed', {
```

Comment:
Are you missing an assertion in this test?

Reply:
Previous test uses something like `analytics.called` to verify if the payload passed was correct.